### DISCUSSION TITLE
This is a test idea

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#280] [General]
In the interest of the community, I suggest we nominate @Maikuolan Viscount on the order of the PHP paradigm.

![img](https://i.postimg.cc/6pXFHJKS/ezgif-com-gif-maker-21.gif)

**Comment by 737simpilot** [comment]
Test Gitter link.


https://gitter.im/CIDRAM/Lobby?at=61e3478482a4667b257a2d8d

**Comment by 737simpilot** [comment]
#274 

Test

**Reply by 737simpilot** [reply]
test

**Reply by 737simpilot** [reply]
test

**Reply by 737simpilot** [reply]
- test 
- test

**Comment by Maikuolan** [comment]
Looks like it's all working properly. Nice. :+1:

![Nice](https://user-images.githubusercontent.com/12571108/150113191-bb5f9315-efe0-4549-99a2-65362530aa32.jpg)
